src/desktop/apps/fair/index.coffee

```diff
@@ -22,20 +23,20 @@ getFairByOrganizerYear = [
 ]
 
 app.use routes.microsite
-app.get '/:id', getFairData, routes.overview
-app.get '/:id/:year([0-9]{4})', getFairByOrganizerYear, routes.overview
-app.get '/:id/overview', getFairData, routes.overview
-app.get '/:id/articles', getFairData, routes.fairArticles
-app.get '/:id/for-you', getFairData, routes.forYou
-app.get '/:id/search', getFairData, routes.search
-app.get '/:id/browse/show/:partner_id', getFairData, routes.showRedirect
-app.get '/:id/browse/artworks/artworks', routes.malformedFilterRedirect
-app.get '/:id/browse', getFairData, routes.browse
-app.get '/:id/browse/*', getFairData, routes.browse
-app.get '/:id/sign_up', getFairData, routes.overview
-app.get '/:id/sign_up/:action', getFairData, routes.overview
-app.get '/:id/capture', getFairData, routes.captureSignup, routes.overview
-app.get '/:id/capture/:action', getFairData, routes.captureSignup, routes.overview
+app.get '/:id', redirectFairRequests, getFairData, routes.overview
```

Comment:
Wondering if `redirectFairRequests` might result in a redirect on a profile/handle that _isn't_ a fair. Is that checked somewhere, or does it rely on this app being mounted last after any other apps (partner profile, etc.) that might respond to the `/:id` type route.

Reply:
Implicitly relies on it being at the bottom of the middleware stack, but we def could an an explicit check/`next()`

I even had that, but thought it overkill. Happy to add it back though!